tests/fixtures/semantic_manifest_yamls/simple_semantic_manifest/semantic_models/user_sm_source.yaml

```diff
@@ -8,10 +8,10 @@ semantic_model:
     alias: dim_users
 
   defaults:
-    agg_time_dimension: ds
+    agg_time_dimension: dsds
 
   dimensions:
-    - name: ds
+    - name: dsds
```

Comment:
Not that it matters much but I'd rather rename this to like `ds_source_date` or something.